### PR TITLE
AM-4355: Deploy Wrapped Asset to Sepolia testnet 

### PR DESCRIPTION
**Pull request by zarifpour**
- [x] Tested on holesky
- [x] Added verification scripts
- [x] Refactored deployment script to accommodate new verification script
- [x] Deployed `t21TON` to sepolia testnet at: https://sepolia.etherscan.io/address/0xc324FD57BCa882F18432abE0fF5A2b8917bfEA24#readProxyContract 

The verification script was added in case the contract is not verified at deployment. If this happens you may set the address in `2_verify_contracts.js` to the proxy address and run the script with: 

```shell
npm run verify:<network>
```

> [!Important]
> Deployment arguments are now configured in `scripts/utils.js` instead of `scripts/1_deploy_contracts.js`

> [!Note]
> I tested an upgrade on holesky, so please ignore the following change:
> ```json
> },
> "allAddresses": [
>   "0x73225F88fEEA4E41Fc67E986a95AC61dd7118866"
> ]
> ```

### CODE REVIEW

File: .openzeppelin/unknown-17000.json [hunk 1]
```diff
@@ -311,7 +330,10 @@
           }
         },
         "namespaces": {}
-      }
+      },
+      "allAddresses": [
+        "0x73225F88fEEA4E41Fc67E986a95AC61dd7118866"
+      ]
```
Comment (zarifpour) [inline .openzeppelin/unknown-17000.json:336]: this happened because I tested an upgrade

File: .openzeppelin/sepolia.json [hunk 2]
```diff
@@ -20,6 +20,11 @@
       "address": "0x4941c4CA5Db4Ef0bAA332648db0b09bD8460ff75",
       "txHash": "0x1282c79e33696706371941a8cb554f0394dd36cb034100b0e9294404840a856f",
       "kind": "transparent"
+    },
+    {
+      "address": "0xc324FD57BCa882F18432abE0fF5A2b8917bfEA24",
+      "txHash": "0x65e7ddda99da4f1a7e3c619629a1d29b39b8a4f7757e4b2f04e6473147caed4d",
+      "kind": "transparent"
```
Comment (zarifpour) [inline .openzeppelin/sepolia.json:27]: as expected, only a new proxy for the wrapped asset was added. no new implementation was detected and deployed.

File: .openzeppelin/unknown-17000.json [hunk 3]
```diff
@@ -19,6 +19,25 @@
...
+      "kind": "transparent"
+    },
+    {
+      "address": "0xaFFC30eDb3698eFdDdf03522EfDB5Bf7465821b1",
+      "kind": "transparent"
+    },
+    {
+      "address": "0x094c7b06Be5c50438d89b2712622C03fAB95AA17",
+      "txHash": "0x4b261473a08acebe9ac7a962379090284fdd0ba666fa96b897fef1937d173651",
+      "kind": "transparent"
```
Comment (zarifpour) [inline .openzeppelin/unknown-17000.json:40]: 4 test deployments occurred on holesky

File: scripts/1_deploy_contracts.js
Comment (zarifpour) [inline scripts/1_deploy_contracts.js:1]: all reusable components moved to utils

Comment (JPMora89) [reply scripts/1_deploy_contracts.js:1]: Makes sense, looks good to me.

**Review by JPMora89 — APPROVED** [review]
lgtm

**Pull request merged**